src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -5038,6 +5038,274 @@
 		</Setter>
 	</xamarin:Style>
 
+	<!-- Default style for Windows.UI.Xaml.Controls.ProgressRing -->
+        <wasm:Style x:Key="ProgressRingEllipseStyle" TargetType="Ellipse">
+	        <Setter Property="Opacity" Value="0" />
+        	<Setter Property="HorizontalAlignment" Value="Left" />
+        	<Setter Property="VerticalAlignment" Value="Top" />
+        </wasm:Style>
+	<wasm:Style TargetType="ProgressRing">
```

Comment:
can you add comments documenting the changes you had to make to the UWP style please

Reply:
Actually there was only 1 change from the default UWP style and it already has a comment here:
[https://github.com/unoplatform/uno/pull/1814/files#diff-c3a19d51c5f941f1e8800bf44e3b7334R5089](https://github.com/unoplatform/uno/pull/1814/files#diff-c3a19d51c5f941f1e8800bf44e3b7334R5089)